app/src/main/java/net/bible/android/view/activity/readingplan/ReadingPlanItemAdapter.kt

```diff
@@ -18,45 +18,43 @@
 
 package net.bible.android.view.activity.readingplan
 
-import net.bible.service.readingplan.ReadingPlanInfoDto
 import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ArrayAdapter
-import android.widget.TwoLineListItem
+import kotlinx.android.synthetic.main.reading_plan_list_single.view.*
+import net.bible.service.db.readingplan.ReadingPlanInformationDB
+
 
 /**
  * nice example here: http://shri.blog.kraya.co.uk/2010/04/19/android-multi-line-select-list/
  *
  * @author Martin Denham [mjdenham at gmail dot com]
  */
-class ReadingPlanItemAdapter(_context: Context, private val resource: Int, _items: List<ReadingPlanInfoDto>):
-		ArrayAdapter<ReadingPlanInfoDto>(_context, resource, _items) {
+class ReadingPlanItemAdapter(_context: Context, private val resource: Int, _items: List<ReadingPlanInformationDB>):
```

Comment:
```suggestion
class ReadingPlanItemAdapter(context: Context, private val resource: Int, items: List<ReadingPlanInformationDB>):
```

Or do you have a reason for these _ prefixes?

Reply:
No reason. They were there from before and I haven't thought of changing. Will do it.